testgrid/specs/os-latest.yaml

```diff
@@ -16,19 +16,19 @@
 - id: ol-79
   name: Oracle Linux
   version: "7.9"
-  vmimageuri: https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-2009.qcow2
+  vmimageuri: http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud.qcow2
```

Comment:
why http?

Reply:
that's weird, literally just "copy link address" from Centos page.